generators/app/templates/python/README.md

```diff
@@ -1,29 +1,65 @@
 # <%= name %> modular service
 
-*<%= name %>* is a Viam modular service that provides <%= api_name_lower %> capabilities
+This module implements the [<%= api_namespace %> <%= api_name_lower %> API](https://github.com/<%= api_namespace %>/<%= api_name_lower %>-api) in a <%= model %> model.
+With this model, you can...
 
-The model this module makes available is <%= model %>
+## Requirements
 
-## Prerequisites
-
-Add instructions here for any prerequisites.
+Add instructions here for any requirements.
```

Comment:
We should use italics or brackets to indicate placeholder text that the module author should change:

```suggestion
_Add instructions here for any requirements._
```

Reply:
While I agree this is an improvement, that's also not how it's currently used in this template, so we should change that throughout